ext/nnstreamer/tensor_filter/tensor_filter_llamacpp.cc

```diff
@@ -500,8 +525,11 @@ void
 TensorFilterLlamaCpp::init_filter_llama ()
 {
   registered = tensor_filter_subplugin::register_subplugin<TensorFilterLlamaCpp> ();
-  nnstreamer_filter_set_custom_property_desc (name, "num_predict", "Number of tokens to predict",
-      "num_gpu_layers", "Number of layers to offload to the GPU", NULL);
+  nnstreamer_filter_set_custom_property_desc (name, "num_predict",
+      "Number of tokens to predict", "num_gpu_layers",
```

Comment:
I think `num_gpu_layers` is not supported here.

Reply:
Thank you. I've updated it.